modules/core/03-connection/keeper/handshake.go

```diff
@@ -74,7 +74,7 @@ func (k Keeper) ConnOpenTry(
 	// generate a new connection
 	connectionID := k.GenerateConnectionIdentifier(ctx)
 
-	if err := k.validateHeight(ctx, clientState.ClientType(), consensusHeight); err != nil {
```

Comment:
I found this function name to vague. This function only makes sense when validating the consensus height stored on the counterparty chain's light client